.travis.yml

```diff
@@ -14,10 +14,8 @@ notifications:
 env:
   matrix:
     - ROS_DISTRO=kinetic ROS_REPO=ros              TEST=clang-format
-    - ROS_DISTRO=kinetic ROS_REPO=ros              UPSTREAM_WORKSPACE=moveit.rosinstall
-    - ROS_DISTRO=kinetic ROS_REPO=ros-shadow-fixed UPSTREAM_WORKSPACE=moveit.rosinstall
```

Comment:
in our last maintainer meeting we said we wanted to keep the melodic branch buildable on 16.04 and kinetic for the foreseeable future. do you think this is still feasible? if so, i'd like to keep these kinetic builds here

Reply:
As there are incompatible API changes in OMPL and KDL, we would need to resort to #ifdefs.
I think that should be possible.

Reply:
Currently, Kinetic builds are blocked by: https://github.com/ros/geometry2/pull/292#issuecomment-390114500.

Reply:
If https://github.com/ros-planning/moveit/pull/830 is merged, it depends on 3 upstream ROS package API breakages:
https://github.com/ros/geometry/pull/163
https://github.com/ros/geometry2/pull/292
https://github.com/ros/geometry2/pull/294
https://github.com/ros-visualization/rviz/pull/1236
I doubt these breaks should/will be backported, which would force us to choose between tf2 migration or keeping things buildable on both kinetic & melodic.

Reply:
It would be a pity in both cases. I asked ROS maintainers to consider all of them for merging in https://github.com/ros/geometry2/pull/292#issuecomment-390235380.

Reply:
If we can't support Melodic on 16.04 i understand, but that will sure slow down adoption I imagine.